tests/coalaJSONTest.py

```diff
@@ -150,3 +151,17 @@ def test_output_file(self):
         self.assertFalse(stdout2)
         self.assertFalse(stderr2)
         os.remove('file.json')
+
+    def test_show_language_bears_output_file(self):
+        with bear_test_module():
+            retval, stdout, stderr = execute_coala(
+                coala.main, 'coala', '--json', '-B', '--filter-by', 'language',
+                'java', '-I', '--output', 'bears.json')
+
+        with open('bears.json') as fp:
+            data = json.load(fp)
+
+        self.assertEqual(retval, 0)
+        self.assertEqual(len(data['bears']), 2)
+        self.assertFalse(stderr)
+        os.remove('bears.json')
```

Comment:
remove the file immediately once you've loaded contents, then the file doesn't remain on test fails (which would cause then test fails, because fo the append mode!)